components/event-bus/cmd/subscription-controller/cmd/manager/main.go

```diff
@@ -68,27 +70,11 @@ func main() {
 		log.Error(err, "unable to add Knative Eventing APIs to scheme")
 		os.Exit(1)
 	}
-	// Setup all Controllers
```

Comment:
You can remove all of the above too, Knative injects everything for you inside the `context`.

Reply:
can you please specify the lines, I'm afraid that I don't know what to remove exactly safely

Reply:
@antoineco Can you please check now?